test/util_test.rb

```diff
@@ -16,6 +16,15 @@ def test_util_shell_escape
     assert_equal test_class.new.escape_for_shell("vm.test.com physical.test.com"), 'vm.test.com\ physical.test.com'
   end
 
+  def test_util_escape_for_cmdexe
+    assert Proxy::Util.instance_methods.include? :escape_for_cmdexe
+
+    test_class = eval "class ProxyUtilTestHelper; include Proxy::Util; end"
+    assert_equal test_class.new.escape_for_cmdexe("Test text &! continues"), ["Test", "text", "&!", "continues"]
+    assert_equal test_class.new.escape_for_cmdexe("Test text&!continues"), ["Test", "text&!continues"]
+    assert_equal test_class.new.escape_for_cmdexe('vm.test.com,"physical.test.com'), ["vm.test.com,\"physical.test.com"]
```

Comment:
does this actually work? The `escape_for_cmdexe` only splits the command into array by empty space. The actual espacing is performed by passing command as an array on line 40 if I understand.